source/release-notes.txt

```diff
@@ -11,6 +11,46 @@ Release Notes
    :class: twocols
 
 
+1.10.0 Changelog
+---------------
+
+*Released October 1, 2024*
+
+New features:
+
+- New bulk conversion UI and API for Query Converter to automatically 
+  convert large numbers of SQL queries, views and stored procedures. 
+  See :ref:`rm-bulk-conversion`.
+- Major performance improvements with up to 5x faster new document 
+  creation. By default, snapshot jobs now create top-level documents 
+  using an improved, non-idempotent bulk insert method.
```

Comment:
(minimalism) 
Let's rewrite a smidge. Something like:
- Snapshot jobs now create top-level documents using a non-idempotent bulk insert method. This improves performance.